### ISSUE TITLE
F# Unit Tests do not show up in Unit Tests Session window when invoked on solution or namespace in Unit Test Explorer

### ISSUE THREAD

**Issue by Booksbaum** [xunit/resharper-xunit#101]
Given a F# solution "XunitResharperFsharp" with a simple module with a unit test:
```F#
module MyNamespace.Tests

open System
open System.IO
open Xunit

[<Fact>]
let ``run test`` () =
    let path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Desktop), "test.txt")
    File.AppendAllText(path, sprintf "%A%s" DateTime.Now Environment.NewLine)

    Assert.True true
```

The Resharper Unit Test Explorer looks like this:
![unittestexplorer](https://cloud.githubusercontent.com/assets/15612932/11479115/99e9d25e-9790-11e5-8d55-1d974ce8b01d.png)

When running the test from the Unit Test Explorer either via the Solution group ("XunitResharperFsharp") or the Namespace group ("MyNamespace") a session is created and the test is run (file is created/modified) BUT the test isn't displayed in the Unit Test Session windows (it's just empty):
![unittestsessions](https://cloud.githubusercontent.com/assets/15612932/11479149/c89512a8-9790-11e5-9dea-f2a997c03e1c.png)
Running the tests by selecting the Module ("Tests"), the method itself ("run test") or the "Run all tests from Solution" command does work (method shows up in the session window). But only when creating a new unit test session. When there's a "faulty" session open it can reuse, the test will run but the session will stay empty.

**Comment by citizenmatt** [comment]
What versions of ReSharper and the xunit plugin are you using? I'm using 10.0.2 EAP2 + 2.3.2 of the plugin, and I can't reproduce this - clicking run from anywhere starts and runs the tests as expected.

**Comment by Booksbaum** [comment]
Current ReSharper version (10.0.1) and the latest xunit plugin version from the resharper extension gallery (2.3.2).  

Maybe it was a bug in ReSharper that was fixed in 10.0.2.
Or maybe there was something wrong with my installation. VS 2k15 Update 1 messed up my VS installation. I'm currently deinstalling everything and then reinstall VS. Will then test if the bug still occurs, and test it with the 10.0.2 EAP. But that might take some time -- a VS installation is always a 	time-consuming adventure....

**Comment by Booksbaum** [comment]
Tested it with a fresh VS 2k15 (w/ update 1) and ReSharper v. 10.0.1 and the xunit plugin v. 2.3.2. Problem was still there.
Then I tried ReSharper v. 10.0.2 EAP 2 and it works.  
  
Therefore: was an error in ReSharper, that is fixed in the current EAP (10.0.2 EAP2).

**Comment by citizenmatt** [comment]
Thanks for the update. Glad it's fixed!
